Add SAML Inline Hook settings for apps API calls (#2631)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/apps/index.md

| digestAlgorithm       | Determines the digest algorithm used to digitally sign the SAML assertion and response                            | String                                               | FALSE    | FALSE  |                                           |
| honorForceAuthn       | Prompt user to re-authenticate if SP asks for it                                                                  | Boolean                                              | FALSE    | FALSE  |                                           |
| idpIssuer             | SAML Issuer ID                                                                                                    | String                                               | FALSE    | FALSE  |                                           |

> | inlineHooks           | Associate the application with SAML inline hooks. Check the [SAML Inline Hooks](/docs/reference/saml-hook/) for details   | String                                       | TRUE     | FALSE  |                                           |

Reviewer: ```suggestion
| inlineHooks           | Associates the application with SAML inline hooks. See the [SAML Assertion Inline Hook Reference](/docs/reference/saml-hook/) for details.   | String                                       | TRUE     | FALSE  |                                           |
```

* If Single Logout is supported by the application and the `slo` object is provided in the request, the `spCertificate` object must be present.
* When you update an application, if you don't specify `slo` or `spCertificate` the existing configuration persists.

> * When you associate the application with `inlineHooks`, you should [create SAML Inline Hooks](/docs/concepts/inline-hooks/#inline-hook-setup) first, then pass the created Inline Hook IDs.

Reviewer: ```suggestion
* When you associate the application with `inlineHooks`, you should [create SAML Inline Hooks](/docs/concepts/inline-hooks/#inline-hook-setup) first, and then pass the created Inline Hook IDs.
```

            "Value"
          ]
        }
      ],
      "inlineHooks": [
        {
          "id": "cal1chgEoc4sEKNlm0g4",
          "_links": {
            "self": {

>               "href": "http://${yourOktaDomain}/api/v1/inlineHooks/cal1chgEoc4sEKNlm0g4",

Reviewer: should the Inline Hook ID be a variable like yourOktaDomain? ${inlineHookId}?
```suggestion
              "href": "http://${yourOktaDomain}/api/v1/inlineHooks/cal1chgEoc4sEKNlm0g4",
```

      "requestCompressed": false,
      "allowMultipleAcsEndpoints": false,
      "acsEndpoints": [],
      "attributeStatements": [],
      "inlineHooks": [
        {
          "id": "cal3ughy17pylLxQB357",
          "_links": {
            "self": {

>               "href": "https://${yourOktaDomain}/api/v1/inlineHooks/cal3ughy17pylLxQB357",

Reviewer: same question as above

	"requestCompressed": false,
        "allowMultipleAcsEndpoints": false,
        "acsEndpoints": [],
        "attributeStatements": [],
        "inlineHooks": [
          {
            "id": "${inlineHookId}",
            "_links": {
              "self": {

>                 "href": "http://${yourOktaDomain}/api/v1/inlineHooks/${inlineHookId}",

Reviewer 2: ```suggestion
                "href": "https://${yourOktaDomain}/api/v1/inlineHooks/${inlineHookId}",
```

>                 "href": "http://${yourOktaDomain}/api/v1/inlineHooks/cal1chgEoc4sEKNlm0g4",

Reviewer: same question as above
